src/cryptonote_core/service_node_list.cpp

```diff
@@ -397,6 +597,13 @@ namespace service_nodes
     info.last_reward_transaction_index = index;
     info.total_contributed = 0;
     info.total_reserved = 0;
+
```

Comment:
`service_node_info() { *this = {}; }` seems limiting. How about we explicitly state what value we want to use by default using *default member initialisers*? Alternatively we should use a proper constructor, that is what they are designed for. (Not sure this is relevant to this PR though.)

Reply:
My view on this. If we had specific values we want to assign by default to the structure, sure, default member initializer works. Otherwise, specifically writing to zero out is noise and this does it in one line.

I'm not very opinionated about this and I'll leave it to your discretion.